scene/resources/scene_format_text.cpp

```diff
@@ -1644,6 +1644,9 @@ Error ResourceFormatSaverTextInstance::save(const String &p_path, const RES &p_r
 			if (index >= 0) {
 				header += " index=\"" + itos(index) + "\"";
 			}
+			else {
```

Comment:
Style wise, this should be `} else {` to match our guidelines.
But I'm not convinced we should add this workaround, I'd prefer a proper fix for what's causing those indexes to be shuffled around.

Reply:
Fair enough. I'm already using custom builds personally, so it's not a big priority to get this merged. Hopefully someone more familiar with the codebase can pick up this torch.